less/mcq.less

```diff
@@ -53,8 +44,12 @@
     display: none;
   }
 
+  &__input {
+    display: none;
```

Comment:
I can't remember why we had opacity rather than display: none. will check with screen readers etc to make sure we're all good.

Reply:
It's because you can't tab onto the inputs if it's display: none